website/docs/cloud-docs/users-teams-organizations/permissions.mdx

```diff
@@ -176,7 +176,7 @@ The "read" permission set is for users who need to view information about the st
 
 ## Organization Permissions
```

Comment:
Since this is a comprehensive list of the org permissions, I think that we'll probably want to add a subheading here for the new selector we're adding that allows you to "View workspaces only" -- what do you think? Since this selector hasn't been shipped yet, I'd be fine with adding it in a follow up PR, post-ship!

Reply:
i don't think i added it because it wasn't in the figma i was looking at -- i can add this now

Reply:
We don't have a design for it, as it was a late addition to the project, but it will be added to the permissions soon, so it is likely something we'll want to document. Thanks for adding it!